packages/admin-panel/src/routes/surveys/surveys.js

```diff
@@ -131,7 +131,7 @@ const SURVEY_FIELDS = {
       options: SERVICE_TYPES,
       setFieldsOnChange: (newValue, currentRecord = null) => {
         const { dhisInstanceCode = 'regional' } = currentRecord
-          ? currentRecord['data_group.config'] ?? {}
+          ? (currentRecord['data_group.config'] ?? {})
           : {};
```

Comment:
Would be tempted to simplify this to…

```ts
const { dhisInstanceCode = 'regional' } = currentRecord?.['data_group.config'] ?? {};
```

…but I presume this was just a change by Prettier, so also happy for you to call it out-of-scope for this PR